### PR TITLE
Adjust log level for counter metrics

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
The counters should use different log levels so that important counters are not dropped.

#### Summary of Changes
Adjusted the log level for counters.

### CODE REVIEW

**Comment by pgarg66** [comment]
This PR only adjusts the counter log level. Still need to adjust it for ```datapoint!```

File: core/src/banking_stage.rs [hunk 1]
```diff
@@ -359,7 +359,7 @@ impl BankingStage {
         debug!("processed: {} ", processed_transactions.len());
         // unlock all the accounts with errors which are filtered by the above `filter_map`
         if !processed_transactions.is_empty() {
-            inc_new_counter_info!(
+            inc_new_counter_warn!(
```
Comment (sagar-solana) [inline core/src/banking_stage.rs]: Isn't this our primary tps counter?

Comment (pgarg66) [reply core/src/banking_stage.rs]: yep, that's why higher log level

Comment (sagar-solana) [reply core/src/banking_stage.rs]: Oh i see.

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

**Comment by pgarg66** [comment]
https://github.com/solana-labs/solana/issues/4147

**Pull request merged**